### ISSUE TITLE
Everything just bugging out?

### ISSUE THREAD

**Issue by SuperStickFigures** [TheXTech/TheXTech#743]
### Describe the issue

I was just messing around with cheat codes and such and then for some strange reason the game removed my transparent timer and the fade outs were really messed up. 

### A type of bug

Native bug

### Version

1.3.6.4

### Version Hash

#282cf3b

### Platform

Windows

### Processor architecture

Not specified / I don't know

### Log file (if presented)

[TheXTech_log_2024_03_28_07_45_47.txt](https://github.com/TheXTech/TheXTech/files/14789309/TheXTech_log_2024_03_28_07_45_47.txt)

### Example Case

the epsiode was The Invasion 2. 

### Recording

https://youtu.be/M7JAOk540P4

**Comment by Wohlstand** [comment]
And, possibly, there is another related issue created, or this "feature" had been discussed at the Discord ... 🤔

**Comment by Wohlstand** [comment]
This bug is related to this: #4

**Comment by SuperStickFigures** [comment]
no no no not that ignore the yoshi storage. In the video, after some time in Pwnhammer Deuce, my timer just stops being transparent and the fade out stops working

**Comment by Wohlstand** [comment]
Oh wait, now I see, I was on phone and I wasn't read carefully. Sounds like the alpha-channel parameter got messed up here, or if you play with OpenGL render, then sounds like a blend mode got being messed at some moment :thinking:

**Comment by Wohlstand** [comment]
Okay, on my end I can't reproduce it...
Plesae try to play this again with `SDL2` and `OpenGL 1+` render modes (you can toggle this in options menu, and on Windows this requires to restart the game). Also, does this bug gets reproduced always or randomly?

**Comment by Wohlstand** [comment]
Actually, I do have the second computer with Windows and AMD Radeon RX550 thing, so, I can try to reproduce on it.

**Comment by SuperStickFigures** [comment]
> Okay, on my end I can't reproduce it... Plesae try to play this again with `SDL2` and `OpenGL 1+` render modes (you can toggle this in options menu, and on Windows this requires to restart the game). Also, does this bug gets reproduced always or randomly?

it just happens randomly yea

**Comment by Wohlstand** [comment]
By the way, try out the latest devel version (1.3.7-dev), and how much chance it will get such mess?

**Comment by SuperStickFigures** [comment]
kk ill try it

**Comment by SuperStickFigures** [comment]
yeah im just trying to do the same thing i did last time to cause it. and im not getting anything with 1.3.7. But again, it does just happen randomly.

**Comment by Wohlstand** [comment]
Just a question: does this still happen? Or it no longer appears for a while?
